mozregression/launchers.py

```diff
@@ -275,9 +275,58 @@ def create_launcher(buildinfo):
     )
 
 
+class FirefoxRegressionProfile(Profile):
+    """Specialized Profile subclass for Firefox"""
+
+    preferences = {  # Don't automatically update the application
+                   'app.update.enabled': False,
+                     # Don't restore the last open set of tabs
+                     # if the browser has crashed
+                   'browser.sessionstore.resume_from_crash': False,
+                     # Don't check for the default web browser during startup
+                   'browser.shell.checkDefaultBrowser': False,
+                     # Don't warn on exit when multiple tabs are open
+                   'browser.tabs.warnOnClose': False,
+                     # Don't warn when exiting the browser
+                   'browser.warnOnQuit': False,
+                     # Don't send Firefox health reports to the production
+                     # server
+                   'datareporting.healthreport.documentServerURI':
+                   'http://%(server)s/healthreport/',
+                     # Only install add-ons from the profile and the
+                     # application scope
+                     # Also ensure that those are not getting disabled.
+                     # see:
+                     # https://developer.mozilla.org/en/Installing_extensions
+                   'extensions.enabledScopes': 5,
+                   'extensions.autoDisableScopes': 10,
+                     # Don't send the list of installed addons to AMO
+                   'extensions.getAddons.cache.enabled': False,
+                     # Don't install distribution add-ons from the app folder
+                   'extensions.installDistroAddons': False,
+                     # Dont' run the add-on compatibility check during start-up
+                   'extensions.showMismatchUI': False,
+                     # Don't automatically update add-ons
+                   'extensions.update.enabled': False,
+                     # Don't open a dialog to show available add-on updates
+                   'extensions.update.notifyUser': False,
```

Comment:
Let's take out most of these addon preferences, people might want to use mozregression to test this sort of functionality.